zerver/lib/actions.py

```diff
@@ -6457,8 +6457,7 @@ def do_unmute_topic(user_profile: UserProfile, stream: Stream, topic: str) -> No
 
 def do_mark_hotspot_as_read(user: UserProfile, hotspot: str) -> None:
     UserHotspot.objects.get_or_create(user=user, hotspot=hotspot)
-    event = dict(type="hotspots", hotspots=get_next_hotspots(user))
-    send_event(user.realm, event, [user.id])
+    check_is_tutorial_finished(user)
```

Comment:
Yes, that's the reason to do `send_event`.

Reply:
So @timabbott, what would `send_event` send after a hotspot is read (here).

Earlier, it was used to send the next unread hotspot.
But, now we have changed the hotspot infrastructure to send all the hotspots at once. 
So that's why I removed the `send_event` line.

Reply:
I think it should send all hotspots that the user should potentially see, which could be the next one in the onboarding set plus an additional drafts one.  

(I think there's no reason that having this new drafts hotspot that is independent of the normal sequence should require any changes to the behavior of the existing sequenced logic).